lib/Config.php

```diff
@@ -31,56 +32,48 @@ private function castToBool($value): bool {
 		return $value === 'true' || $value === true;
 	}
 
-	/**
-	 * @param string|bool $value
-	 * @return string
-	 */
-	private function castToString($value): string {
-		return ($value === 'true' || $value === true) ? 'true' : 'false';
-	}
-
 	public function allowExternalStorage(): bool {
-		return $this->castToBool($this->config->getAppValue('guests', 'allow_external_storage', 'false'));
+		return $this->castToBool($this->appConfig->getAppValueBool('allow_external_storage', false));
```

Comment:
```suggestion
		return $this->appConfig->getAppValueBool('allow_external_storage', false);
```
No cast is needed anymore

Reply:
I was afraid to remove it for legacy purposes :see_no_evil: